projects/github-actions/test-results-to-slack/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Send tests results to Slack"
+description: "Send Slack notifications with test results."
+branding:
+  icon: 'activity'
+  color: 'green'
+inputs:
+  github_token:
+    description: "GitHub access token"
+    required: true
+    default: ${{ github.token }}
+  slack_token:
+    description: "Slack Bot access token"
+    required: true
+    default: ""
+  slack_channel:
+    description: "Slack channel ID where to send messages"
+    required: true
+    default: ""
+  slack_username:
+    description: "Custom username to be used for messages"
+    required: true
+    default: "Test reporter"
+  slack_icon_emoji:
+    description: "Emoji to use as the icon for this message."
+    required: true
```

Comment:
Those 2 are marked as required, yet we do not use them in our own workflow. Are they?

Reply:
Not at this moment. They're falling back to their default values. We will probably overwrite the emoji in the next phase, based on the test status.